admin-center/display/govbox-federal-register.md

```diff
@@ -1,14 +1,23 @@
 ---
-layout: post
+layout: page
 title: Help Searchers Find Federal Register Documents
 category: manual
 tags: help-manual display modules
+date: December 17, 2014
+last_modified_at: December 17, 2014
+sidenav: admin_center
+redirect_from:
+    - /manual/govbox-federal-register.html
 ---
 
+Find it in the Admin Center: [Search.gov Home]({{ site.baseurl }}/index.html) > [Admin Center](https://search.usa.gov/sites/) > YourSite > Manage Display > Display Overview
+
 Federal agencies publish notices, proposed rules, and final rules in the [*Federal Register*](https://www.federalregister.gov). To make it easier for citizens and communities to understand the regulatory process and to participate in government decision-making, we created the *Federal Register* module to help searchers find these notices and rules.
 
 This module offers direct access to *Federal Register* documents. We pull the data from the [Federal Register API](https://www.federalregister.gov/learn/developers) to provide timely information across all documents.
 
 If you [opt to show the Federal Register]({{ site.baseurl }}/manual/display-overview.html) module, searchers on your site can readily find the notices and rules that your agency has published in the *Federal Register*, as shown in the sample below for a search on [*autoimmune diseases* on NIH.gov](https://search.nih.gov/search?affiliate=nih&query=autoimmune+diseases).
 
-[![Federal Register Module on NIH.gov](https://d3qcdigd1fhos0.cloudfront.net/blog/img/govbox-fr.png "Federal Register Module on NIH.gov")](https://search.nih.gov/search?affiliate=nih&query=autoimmune+diseases)
\ No newline at end of file
+[![Federal Register Module on NIH.gov](https://d3qcdigd1fhos0.cloudfront.net/blog/img/govbox-fr.png "Federal Register Module on NIH.gov")](https://search.nih.gov/search?affiliate=nih&query=autoimmune+diseases)
+
+If you would like to show [*Federal Register*]({{ site.baseurl }}/manual/govbox-federal-register.html)documents in your results, but do not see the toggle to turn them on, [email us](mailto:search@support.digitalgov.gov) for assistance.
```

Comment:
same as for the jobs page, realized that the link here points back to this same page. please remove.